packages/core/src/ceramic.ts

```diff
@@ -237,93 +284,107 @@ class Ceramic implements CeramicApi {
   }
 
   /**
-   * Returns a copy of the given CeramicConfig object but with any potentially sensitive fields
-   * removed so that it is safe to log the whole thing.
-   * @param config
-   * @returns Copy of `config` with potentially sensitive information removed
-   * @private
-   */
-  private static _redactConfigForLogging(config: CeramicConfig): CeramicConfig {
-    const redactedConfig = {...config}
-    delete redactedConfig.didProvider
-    return redactedConfig
-  }
-
-  /**
-   * Create Ceramic instance
-   * @param ipfs - IPFS instance
-   * @param config - Ceramic configuration
+   * Parses the given `CeramicConfig` and generates the appropriate `CeramicParameters` and
+   * `CeramicModules` from it. This usually should not be called directly - most users will prefer
+   * to call `Ceramic.create()` instead which calls this internally.
    */
-  static async create(ipfs: IpfsApi, config: CeramicConfig = {}): Promise<Ceramic> {
-    // todo remove
+  static async _processConfig(ipfs: IpfsApi, config: CeramicConfig): Promise<[CeramicParameters, CeramicModules]> {
```

Comment:
@stbrody As it can be called by a consumer, this better not have a leading underscore.

Reply:
`_init` will go away, but this will remain, so better rename it now.

Reply:
So I did this on purpose, but it is admittedly an unusual/controversial choice.  Basically I was thinking of `_processConfig` and `_init` as kind of "semi-private" functions.  They're not technically private so they *can* be called by a consumer, but most of the time they *shouldn't*.  So the leading underscore is a signal to consumers that these aren't really part of the standard API

Reply:
I'm going to go ahead and merge this since it's ready to go, but I have another PR coming up that touches this code so I can change it there if you'd like me to drop the underscore, I don't feel super strongly either way

Reply:
Well, let it be then along with _init, and let's wait for complaints, if they ever appear.